src/ToData.purs

```diff
@@ -1,6 +1,21 @@
 module ToData
```

Comment:
Are you going to try to tackle generic `FromData` as well in this PR?

Reply:
Done :)